src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -39,17 +41,17 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
     })
   }, [])
 
+  useEffect(() => {
+    if (forceValidation) onUpdate(currentResponse?.text)
+  }, [currentResponse, forceValidation])
+
```

Comment:
I don't see what is the difference with using this or not. In the code you marked it is re-running the validation with the current response text, and this addition would only set the validation on the first time the plugin is mount, never again. So what is the point of setting this there (each `ApplicationView` is already calling the `onUpdate` when the mount, so that would already set the validation - from my point of view).

Reply:
Yeah, I think that's right. I had thought that the saved "isValid" state was not being sent to the plugins if the value was "undefined", but that appears not to be the case now, so we can omit this for now.